src/SmoFacade/Server.cs

```diff
@@ -59,10 +59,10 @@ public void DropLogin(LoginProperties login)
 
     public decimal DatabaseSizeMb(string dbName)
     {
-      string query = "SELECT total_size_mb = (SUM(size) * 8. / 1024) " + 
-                     "FROM sys.master_files WITH(NOWAIT) " +
-                     $"WHERE database_id = DB_ID('{dbName}') " + 
-                     "GROUP BY database_id";
+      var query = "SELECT total_size_mb = (SUM(size) * 8. / 1024) " +
+                  "FROM sys.master_files WITH(NOWAIT) " +
+                  $"WHERE database_id = DB_ID('{dbName}') " +

```

Comment:
Can we parameterize dbName since it's a string value?

Reply:
yep good point will change that